### ISSUE TITLE
roca-detect-tls throws: ModuleNotFoundError: No module named 'detect'

### ISSUE THREAD

**Issue by Marcool04** [crocs-muni/roca#43]
Hi there...
Seems I've come across a bit of a bug in the roca-detect-tls code.
I am getting the same error with both `pip install roca-detect` and an installation done also with pip from a local github clone. Here it the output:

````
$roca-detect-tls ./tls-list.txt
Traceback (most recent call last):
  File "/usr/bin/roca-detect-tls", line 11, in <module>
    load_entry_point('roca-detect==1.2.5', 'console_scripts', 'roca-detect-tls')()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 570, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2755, in load_entry_point
    return ep.load()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2409, in load
    return self.resolve()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2415, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/lib/python3.6/site-packages/roca/detect_tls.py", line 23, in <module>
    from detect import RocaFingerprinter, logger, LOG_FORMAT
ModuleNotFoundError: No module named 'detect'
````

````
$cat ./tls-list.txt
localhost:443
````

````
$uname -a
Linux 4.9.56-1-lts #1 SMP Thu Oct 12 22:34:15 CEST 2017 x86_64 GNU/Linux
$python -V
Python 3.6.2
$pip -V
pip 9.0.1 from /usr/lib/python3.6/site-packages (python 3.6)
````

**Comment by ph4r05** [comment]
Thanks! Will check that out today

**Comment by ph4r05** [comment]
fixed in v1.2.6

**Comment by Marcool04** [comment]
Awsomely quick fix! Verified working on same setup as initial report.
Thanks a lot, and keep up the good work on the security front!
All the best,
Mark.
